photon-client/src/main/scala/com/linkedin/photon/ml/data/avro/ResponsePredictionFieldNames.scala

```diff
@@ -17,12 +17,7 @@ package com.linkedin.photon.ml.data.avro
 /**
  * Response prediction format fields names.
```

Comment:
Yeah - documenting why we have these two different (but very similar) `AvroFieldNames` derived classes. Should we document it? Should we move them to some LinkedIn-internal codebase and a LinkedIn-internal Driver?